pset-iterators.js

```diff
@@ -92,7 +102,13 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentialParties = presidents.filter(function (parties) {
+  return parties.party
+}).map(function (parties) {
+  return parties.party
+})
 
+console.log(presidentialParties)
```

Comment:
@OcielC Good Jobe here getting to this solution here. I see that you are filtering the president's party and mapping to return only the president parties.
One suggestion for this example you could've used just the `map` method to get your desired outcome.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)